### DISCUSSION TITLE
Welcome to Magnifier Discussions!

### DISCUSSION THREAD

**Discussion by davwat** [CluckCluckChicken/Magnifier#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * go away

  To get started, get mogus drip and become epic gamer.

**Comment by JackyBorderCollie** [comment]
yesssssss and pls make a channel which is named aMoGuS dRiP aNd ePiC gAmErS!11!!!!!1!!

**Comment by FunctionalMetatable** [comment]
we are gamers

**Reply by JackyBorderCollie** [reply]
yessssssssssssssss we gamers

**Reply by davwat** [reply]
epic gamers
